src/api/block.rs

```diff
@@ -55,7 +55,6 @@ impl BlockApi {
         &self,
         req: BlockTransactionRequest,
     ) -> ApiResult<BlockTransactionResponse> {
-        // TODO Should verify that index is correct?
```

Comment:
The comment was for the index/height `req.block_identifier.index` which for whatever reason is non-optional for this call. I.e. checking if the height and hashes are consistent. Doing this would take an extra call (to `get_block_info`) and I'm unsure if that is worth it compared to just ignoring the index. :shrug:

Reply:
I'll just document that the field is ignored.